.travis.yml

```diff
@@ -1,22 +1,59 @@
 language: python
 python:
 - 2.7
+- 3.5
 - 3.6
```

Comment:
Maybe skip the 3.6 tests for now?  I really don't want to use 38 Travis workers for each PR on this repo, 22 is slightly more reasonable.